src/components/Modals/FiatRamps/fiatRampProviders/banxa.ts

```diff
@@ -10,6 +10,7 @@ export const getBanxaAssets = () => {
     // name will be set in useFiatRampCurrencyList hook
     name: '',
   }))
+  console.log(assets)
```

Comment:
i know this is draft, we'll need to use a `moduleLogger` now as we just banned `console.*` methods in this repo

Reply:
Sure, this is for debugging only